server/app/boards.go

```diff
@@ -20,6 +20,18 @@ var (
 	ErrInsufficientLicense    = errors.New("appropriate license required")
 )
 
+const linkBoardMessage = "@%s linked Board [%s](%s) with this channel"
+const unlinkBoardMessage = "@%s unlinked Board [%s](%s) with this channel"
+
+//todo: remove and use common error
+type BoardNotFoundErr struct {
```

Comment:
Error vars should begin with `Err`  e.g. ErrBoardNotFound   Also it would be useful to have this wrap a `ErrNotFound` so that `model.IsErrNotFound` would be true.